charts/kubernetes-external-secrets/templates/rbac.yaml

```diff
@@ -46,4 +46,22 @@ subjects:
   - name: {{ template "kubernetes-external-secrets.serviceAccountName" . }}
     namespace: {{ .Release.Namespace | quote }}
     kind: ServiceAccount
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
can you add this to https://github.com/godaddy/kubernetes-external-secrets/blob/master/external-secrets.yml?

Reply:
Adding!